### PR TITLE
PRS13865 to 867

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Persons related to Abb 58 and MY010.

### CODE REVIEW

File: new/PRS13868GabraMadhen.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    <persName type="alt"><roleName type="title">mamǝre</roleName> Gabra Mikāʾel</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13868GabraMadhen" passive="ESmy010"/>
+                    <relation name="iha:Place_of_activity" active="PRS13868GabraMadhen" passive="INS0159QQM"/>
```
Comment (eu-genia) [inline new/PRS13868GabraMadhen.xml:59]: please never use IHA relations. I only added them to validate the records, but they are explicitly described in the schema is only to be used by IslHornAfr project

**Review by eu-genia — APPROVED** [review]

File: new/PRS13868GabraMadhen.xml (hunk 1 above)
Comment (eu-genia) [inline new/PRS13868GabraMadhen.xml:59]: the information should move to `floruit`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13868GabraMadhen.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13868GabraMadhen.xml:59]: I tried.

Comment (DenisNosnitsin1970) [reply new/PRS13868GabraMadhen.xml:59]: ok.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**